packages/mariadb-10.11-mroonga/debian/rules

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/make -f
+# -*- makefile-gmake -*-
+#
+# Uncomment this to turn on verbose mode.
+#export DH_VERBOSE=1
+# This has to be exported to make some magic below work.
+export DH_OPTIONS
+
+export MARIADB_VERSION_FULL := $(shell apt-cache show mariadb-server | grep Version | head -n 1 | awk '{print $$2}' | cut -d: -f2)
+export MARIADB_VERSION := $(shell apt-cache show mariadb-server | grep Version | head -n 1 | awk '{print $$2}' | cut -d: -f2 | awk -F '-' '{print $$1}')
+
+%:
+	dh $@
+
+override_dh_auto_configure:
+	archive_base=mariadb_$(MARIADB_VERSION);			\
+	archive=$${archive_base}.orig.tar.gz;				\
+	debian=mariadb_$(MARIADB_VERSION_FULL).debian.tar.xz;	\
+	source_base=mariadb-$(MARIADB_VERSION);				\
+	build_dir=$${source_base}/builddir;				\
+	if [ "$$(lsb_release --id --short)" = "Ubuntu" ]; then		\
+	  path=universe/m/mariadb-10.11;					\
+	  base_url=http://archive.ubuntu.com/ubuntu/pool;		\
+	  security_base_url=http://security.ubuntu.com/ubuntu/pool;	\
+	else								\
+	  path=main/m/mariadb;					\
+	  base_url=http://ftp.debian.org/debian/pool;			\
+	  security_base_url=http://security.debian.org/pool/updates;	\
+	fi;								\
+	wget $${security_base_url}/$${path}/$${archive} ||		\
+	  wget $${base_url}/$${path}/$${archive};			\
+	wget $${security_base_url}/$${path}/$${debian} ||		\
+	  wget $${base_url}/$${path}/$${debian};			\
+	tar xf $${archive};						\
+	(cd $${source_base} &&						\
+	  tar xf ../$${debian});					\
+	dpkg-source --before-build $${source_base};			\
+	(cd $${source_base} &&						\
+	  debian/rules override_dh_auto_configure);			\
+	(cd $${build_dir}/storage/perfschema &&				\
+	  $(MAKE) -j$$(nproc) perfschema);				\
+	dh_auto_configure --						\
+	  --with-mysql-source=$${source_base}				\
+	  --with-mysql-build=$${build_dir}				\
+	  --with-mysql-config=$${build_dir}/scripts/mysql_config
```

Comment:
Can we use CMake by using `dh_auto_config --buildsystem=cmake+ninja -- ...`?
(We can do it as a follow-up PR.)